package/src/contexts/messageInputContext/MessageInputContext.tsx

```diff
@@ -1039,26 +994,30 @@ export const MessageInputProvider = ({
           if (message && isBouncedMessage(message)) {
             await removeMessage(message);
           }
+
           value.sendMessage({
             attachments,
-            mentioned_users: uniq(mentionedUsers),
+            // TODO: Handle unique users
+            mentioned_users: localMessage.mentioned_users?.map((user) => user.id),
```

Comment:
don't forget the `TODO`

Reply:
This is handled in attachments manager PR so please bear with it for now